### ISSUE TITLE
Missing instructions to install dependencies

### ISSUE THREAD

**Issue by pattacini** [icub-tech-iit/icub-setup-installation#1]
Hi @Nicogene 

The following installation:

https://github.com/icub-tech-iit/icub-setup-installation/blob/39aa2a744dba0416da52403abe734da1562c8a93/.gitpod.Dockerfile#L26

can be actually replaced on the laptop (in the Dockerfile too) by doing:
```sh
$ sudo apt install -y icub-common
```

I don't think you mentioned in the README to install `icub-common`. If so, you ought to clearly say that all those dependencies need to be installed anyway.

However, I'd honestly opt for installing `icub-common` instead, given the ultimate goal of this guide.

cc @traversaro

**Comment by pattacini** [comment]
Also, installing dependencies should be the very first step to be done in the assignment.
Unfortunately, Gitpod doesn't allow `sudo` from the terminal; hence, we're forced to pre-install them in the Docker image.

You may want to say that.

**Comment by pattacini** [comment]
I believe this issue has been addressed by the latest commits.
Isn't it @Nicogene?

**Comment by traversaro** [comment]
Ack for using `icub-common` in these instructions. Just for context, the reason why historically I did not used it in the robotology-superbuild mainly for three reasons.
The first one is that the code for installing the repo, i.e. : 
~~~
sh -c 'echo "deb http://www.icub.org/ubuntu focal contrib/science" > /etc/apt/sources.list.d/icub.list'
apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 57A5ACB6110576A6
apt update
apt install -y icub-common
~~~
is Ubuntu Focal specific, the second one is that in the past icub-common was frequently outdated w.r.t. to dependencies, and the third one is that in Homebrew we did not have any "meta-package" just to list dependencies, so for consistency I did the same for Debian-based instructions.
